File: uri-judge/[2006]Identificando_O_Cha.cpp
//Author: Hrz - 2ºSM de CC
//Site: hrezend.github.io

#include <bits/stdc++.h>
#define desync ios_base::sync_with_stdio(false);cin.tie(NULL);cout.tie(NULL)
#define endl '\n'
using namespace std;

int main(){

	desync;
	int Tipo;
	int Palpites[5];
	int Contador = 0;

	cin >> Tipo;
	
	for(int i = 0; i < 5; i++){
		cin >> Palpites[i];
		if(Palpites[i] == Tipo){
			Contador++;
		}	
	}
	
	cout << Contador << endl;

}

File: uri-judge/java-resolutions/[2006]Identificando_O_Cha.java
//Author: Hrz - 2ºSM de CC
//Site: hrezend.github.io

import java.io.IOException;
import java.util.Scanner;

public class Main {
 
    public static void main(String[] args) throws IOException {
    
        Scanner hr = new Scanner(System.in);
        int Tipo;
	    int Palpites[] = new int [10];
	    int Contador = 0;
	    
	    Tipo = hr.nextInt();
	    
	    
	    for(int i = 0; i < 5; i++){
		    Palpites[i] = hr.nextInt();
    		if(Palpites[i] == Tipo){
		    	Contador++;
	    	}	
	    }
    
        System.out.println(Contador);
 
    }
 
}
